### ISSUE TITLE
Azure Snowflake

### ISSUE THREAD

**Issue by theianrobertson** [capitalone/Data-Load-and-Copy-using-Python#32]
Snowflake can run off azure now (and gcp sometime this year I think), so we should at least add in the docs that we don't support azure/gcp, and maybe look at supporting them (though that could be tricksy with testing)

**Comment by fdosani** [comment]
Agreed! I'll add to the docs on the next set of updates when I have time.
